source/release_policy.html.erb

```diff
@@ -19,8 +19,20 @@ description: Solidus follows semantic versioning and tries to keep a cadence for
       </p>
 
       <p class="lead">
-        Solidus is in strict compliance with <a href="https://semver.org/">Semantic Versioning</a>.
-      </p>
+        Solidus is in strict compliance with <a
+          href="https://semver.org/">Semantic Versioning</a>. Our definition of a
+        Breaking Change is any update to the Solidus code that can break a host
+        application, except for:
+      </p>
+
+      <ul>
+        <li>Issues due to the application using an interface explicitly marked as private.</li>
+        <li>Issues due to the application monkey patching Solidus code.</li>
+        <li>Issues due to a fix for a security or very important bug for which
+          it hasn't been possible to implement a non-breaking solution.</li>
+        <li>Issues due to the application using a Ruby/Rails version that is no
```

Comment:
@waiting-for-dev Have you considered expanding the scope of this exception to _any_ dependency of Solidus that is no longer maintained?

Reply:
Good call. What do you think about the update?

Reply:
@waiting-for-dev Awesome, thanks!